.assistance/Vagrantfile

```diff
@@ -45,9 +45,9 @@ Vagrant.configure("2") do |config|
     echo "export GOLANG_VERSION=1.14" >> /home/vagrant/.env
     echo "export RVM_VERSION=1.29.12" >> /home/vagrant/.env
     echo "export RUBY_VERSION=2.4.2" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_USER=user" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_PASSWORD=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_URL=artifactory.stackstate.io/artifactory/api/pypi/pypi-local" >> /home/vagrant/.env
+    echo "export artifactory_user=$artifactory_user" >> /home/vagrant/.env
+    echo "export artifactory_password=$artifactory_password" >> /home/vagrant/.env
```

Comment:
Wouldn't this just assign existing ENV vars back to themselves?

Reply:
this is not setting back to itself, it is setting it in the vagrant env file
the vagrant vm does not inherit your local env variables